introduction.md

```diff
@@ -2,3 +2,9 @@
 Let me introduce myself. I am Charles Milam from Austin, TX.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?